Dockerfile

```diff
@@ -95,3 +80,16 @@ RUN /swift-tensorflow-toolchain/usr/bin/swift build -c release
 
 WORKDIR /open_spiel
 RUN /swift-tensorflow-toolchain/usr/bin/swift test
+
+WORKDIR /swift-apis
+# TODO: move into bash scripts...
+RUN rm -f /swift-tensorflow-toolchain/usr/lib/swift/linux/x86_64/TensorFlow.swiftinterface
+RUN rm -f /swift-tensorflow-toolchain/usr/lib/swift/linux/x86_64/TensorFlow.swiftdoc
+RUN rm -f /swift-tensorflow-toolchain/usr/lib/swift/linux/x86_64/TensorFlow.swiftmodule
+RUN rm -f /swift-tensorflow-toolchain/usr/lib/swift/linux/libswiftTensorFlow.so
```

Comment:
Are these rm's still necessary?

Reply:
The command above does an install.

Reply:
Oh I see, `-D CMAKE_INSTALL_PREFIX=/swift-tensorflow-toolchain/usr` tells it to put the stuff here.

Reply:
@marcrasi yeah, that caught me too.  Perhaps this should get commented?’

Reply:
This is testing that swift-pm works against the installed cmake build. They can go if the swift-pm goes.